### PR TITLE
Add Validation to Login Form

### PR DESCRIPTION
**Pull request by DimaUstim**

### CODE REVIEW

File: src/app/service/sing-in/sing-in.service.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ export class SingInService {
   constructor() { }
 
   login(data?: User) {
-
```
Comment (AndriiPanasenkoSolvve) [inline src/app/service/sing-in/sing-in.service.ts]: Redundant changes

File: src/app/model/user/user.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export interface User {
     email?: string;
+    password?: string;
```
Comment (AndriiPanasenkoSolvve) [inline src/app/model/user/user.ts]: What is a purpose of keeping the password?

File: src/app/components/sing-in/sing-in.component.html [hunk 3]
```diff
@@ -1,19 +1,76 @@
...
+      >
+    </div>
   </div>
-  <div class="mt-3 mb-3">
+  <div class="mt-2 mb-3">
     <p-button
-      (click)="onBtnClick()"
-      (click)="close()"
+      (onClick)="onBtnClick()"
+      (onClick)="close()"
```
Comment (AndriiPanasenkoSolvve) [inline src/app/components/sing-in/sing-in.component.html]: It is better to use one (onClick) callback and move all logic in it

File: src/app/components/sing-in/sing-in.component.html [hunk 4]
```diff
@@ -1,19 +1,76 @@
...
+        *Name must be at least 6 characters long</small
+      >
+    </div>
   </div>
-  <div class="mt-3 mb-3">
+  <div class="mt-2 mb-3">
     <p-button
-      (click)="onBtnClick()"
-      (click)="close()"
+      (onClick)="onBtnClick()"
```
Comment (AndriiPanasenkoSolvve) [inline src/app/components/sing-in/sing-in.component.html]: Try to use function names that make more sense. As a function does something it is a good idea to put a verb in its name that describes what exactly the function does.

**Review by AndriiPanasenkoSolvve — CHANGES_REQUESTED** [review]
Fix minor code styling issues please

File: src/app/service/sing-in/sing-in.service.ts (hunk 1 above)
Comment (DimaUstim) [reply src/app/service/sing-in/sing-in.service.ts]: Fixed

File: src/app/model/user/user.ts (hunk 2 above)
Comment (DimaUstim) [reply src/app/model/user/user.ts]: Fixed

File: src/app/components/sing-in/sing-in.component.html (hunk 3 above)
Comment (DimaUstim) [reply src/app/components/sing-in/sing-in.component.html]: Fixed

File: src/app/components/sing-in/sing-in.component.html (hunk 4 above)
Comment (DimaUstim) [reply src/app/components/sing-in/sing-in.component.html]: Fixed

**Review by AndriiPanasenkoSolvve — APPROVED** [review]

**Review by anastasiia-perederii — APPROVED** [review]

**Pull request merged**